k8s/postgresql/README.md

```diff
@@ -16,9 +16,12 @@ Popular open source software stacks on Kubernetes packaged by Google and made av
 
 This solution will install single instance of PostgreSQL server into your Kubernetes cluster.
 
-The PostgreSQL Pod is managed by a ReplicaSet with the number of replicas set to one. The PostgreSQL Pod uses a Persistent Volume to store data, and a LoadBalancer Service to expose the database port externally. Communication between client and server is encrypted. If you need to limit access to the PostgreSQL instance, you must configure GCP firewall rules.
+The PostgreSQL Pod is managed by a ReplicaSet with the number of replicas set to one. The PostgreSQL Pod uses
```

Comment:
Could you update diagram as well, please?

Reply:
Done.